JS/night.js

```diff
@@ -0,0 +1,30 @@
+// // NIGHT MODE
+
+let nightButton = document.querySelector('.nightMode')
```

Comment:
For all the variables that are never reassigned, we should declare them as `const`